### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by Lythenas**
Add two issue templates: bugs and feature requests.

See it in action here: https://github.com/Lythenas/PaperWM/issues/new/choose

Also added a script to make it easier to gather the system information that might be useful to us:
```
$ ./gather-debug-info.sh
# Please include this information in your bug report on GitHub!
Distribution: Arch Linux
GNOME Shell 43.3
PaperWM branch/tag: develop
PaperWM commit: 223ff883bca9bf20dbf066eceda891cb6e8be931
Enabled extensions:
- paperwm@hedning:matrix.org
- switcher@landau.fi
- dash-to-panel@jderose9.github.com
- appindicatorsupport@rgcjonas.gmail.com
```

For bug reports we could also use an issue for (which is a beta feature):
- more information here: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
- IMO good example here: https://github.com/doomemacs/doomemacs/issues/new/choose

But I think for our purposes this is sufficient for now.

Also I still allowed to start with an empty issue template.

### CODE REVIEW

**Comment by jtaala** [comment]
dang this looks good @Lythenas!  Yes please.  @smichel17 what do you think?  _Also, just following up - if @Lythenas isn't a contributor yet (e.g. can review) - please add him!_

**Comment by Lythenas** [comment]
> _Also, just following up - if @Lythenas isn't a contributor yet (e.g. can review) - please add him!_

I was invited on the weekend but haven't tried reviewing anything yet.

**Review by jtaala — APPROVED** [review]
Like it a lot.  Gather info script works great and gives pretty much what I'd expect we need to debug etc.

**Comment by jtaala** [comment]
@Lythenas - approved.  Looks very good to me! - will leave it to you to merge in when you're ready.

Thanks @Lythenas.

**Comment by Lythenas** [comment]
Ah it seems like I am not authorized to merge... So if you can, feel free to merge this.

@smichel17 maybe take a look if this is the intended behavior. The approval shows as green, but merging is still blocked with "The base branch restricts merging to authorized users". If this is intended it's fine. But I feel like normally the PR creator should merge after getting approval.

**Comment by jtaala** [comment]
> Ah it seems like I am not authorized to merge... So if you can, feel free to merge this.
> 
> @smichel17 maybe take a look if this is the intended behavior. The approval shows as green, but merging is still blocked with "The base branch restricts merging to authorized users". If this is intended it's fine. But I feel like normally the PR creator should merge after getting approval.

Yeah, agree - I assumed that would be the case btw.  I merged it in any case.

**Comment by jtaala** [comment]
@Lythenas!  This has been the best things since sliced bread to help debug issues.  Can we also add to the `gather-system-info.sh` script something to output if users are using wayland or x11?

Happy to create a new issue for this if you want me to.

**Comment by Lythenas** [comment]
@jtaala I created a new PR for this #512

**Pull request merged**
